ena-submission/scripts/ena_submission_helper.py

```diff
@@ -129,6 +129,18 @@ def get_project_xml(project_set):
         "PROJECT": dataclass_to_xml(project_set, root_name="PROJECT_SET"),
     }
 
+def reformat_authors(authors: str) -> str:
+    """This function reformats the Loculus authors string to the format expected by ENA"""
+    authors_list = [author for author in authors.split(";") if author]
```

Comment:
What's the input format here? The one we show as authors on ppx?
`Nicole M Feriancek, Mallery I Breban, Ellie Bourgikos, Angela Bransfield, Philip M Armstrong, Chantal B.F. Vogels, Nathan D Grubaugh`

In that case splitting by `;` doesn't make sense? It must be something else. Maybe would be good to be more explicit what the expected input format is!

Reply:
? This PR changes how we store authors in Loculus/PP so ena-submission will nolonger get authors in the format 
`Nicole M Feriancek, Mallery I Breban, Ellie Bourgikos, Angela Bransfield, Philip M Armstrong, Chantal B.F. Vogels, Nathan D Grubaugh` but in the format `Feriancek, Nicole M;` etc. If the ena-submission pod gets authors in an unexpected format it will throw an error and not submit the manifest

Reply:
I'm confused why you are asking as you updated the function comment to include the expected format e.g. `John A. Doe; Jane B. C. Roe;`

Reply:
Sorry, I hadn't gotten to that part yet